src/components/common/GraphsPlaceholder/GraphsPlaceholder.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Carousel } from 'antd';
+import graphA from '../../../assets/graph-a.jpg';
+import graphB from '../../../assets/graph-b.jpg';
+import graphC from '../../../assets/graph-c.jpg';
+
+export default function GraphsPlaceholder() {
+  // let graphs = [graphA, graphB, graphC]
```

Comment:
Normally I'd say we don't need commented out code but it looks like this will be implemented later when the graphs are pulled in. So not too big of a deal.

Reply:
I've resolved the issue.